backend/src/folder.ts

```diff
@@ -0,0 +1,89 @@
+import { APIGatewayEvent, Context, ProxyResult } from 'aws-lambda'
+const Folder = require('../model/Folder')
+
+const mockUserid = 1
+const mockCourseId = 1
+
+const getFolder = async (): Promise<ProxyResult> => {
+    return Folder.findAll({
+        where: {
+            userId: mockUserid,
```

Comment:
get folder for courseId, not userId. professor could have multiple courses

Reply:
Both courseId and userId for access permission.